package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "iframe-coordinator-monorepo",
-  "version": "5.5.4",
+  "version": "5.5.6",
```

Comment:
The package version also got out of sync with what's published somewhere along the way, so I bumped it in the repo to clear that out.